Sources/PubNubChatComponents/Theming/Reaction/ReactionTheme.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ReactionTheme.swift
+//  PubNubChat
+//
+//  Created by Maciej Adamczyk on 31/08/2022.
+//
+
+import UIKit
+
+public struct ReactionTheme {
```

Comment:
Should it be commented for external developer?

Reply:
It could be, but I think you don't have to do it now. There's a [JIRA issue](https://pubnub.atlassian.net/browse/UI-1292) in order to provide more inline documentation in Swift files. Another approach would be aligning with Karolina so we could immediately while working on new features.

Reply:
Before I see your comment I added some comments, If it is not ok then I will remove all of them @jguz-pubnub

Reply:
It's fine, let's keep it. We can improve it if needed while providing more in-source documentation in the future